2-positioning/positioning.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="es">
+    <head>
+    <meta name="og:title" property="og:title" content="Positionig">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Posicionamiento</title>
+    </head>
+    <body>
+    <header>
+    <h1>This is my first layout</h1>
+    </header>
+    <nav>
+        <ol>
```

Comment:
```suggestion
        <ul>
```
Usually, nav items use unsorted lists.